robber/matchers/contain.py

```diff
@@ -5,16 +5,33 @@
 
 class Contain(Base):
     """
-    expect({'key': value}).to.contain('key')
-    expect([1, 2, 3]).to.contain(2)
+    expect({'key': value}).to.contain('key 1', 'key 2', 'key n')
+    expect([1, 2, 3]).to.contain(1, 2, 3)
     """
 
     def matches(self):
-        return self.expected in self.actual
+        expected_list = list(self.args)
+        expected_list.insert(0, self.expected)
+
+        if not self.is_negative:
+            for expected in expected_list:
+                if expected not in self.actual:
+                    self.expected_arg = expected
+                    return False
+
+            return True
+        else:
+            # As this is the negative case, we have to flip the return value.
+            for expected in expected_list:
```

Comment:
@hieueastagile I have to get the included/excluded elements so `all` and `any` just don't work. I end up using set's `difference` and `intersection`. It's easy to read too.